d2l-table-style.html

```diff
@@ -0,0 +1,309 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+
+<!--
+This module styles table elements under <content> tags.
+
+	<style include="d2l-table-style"></style>
+	<table class="d2l-table">
+		<template>
+			<content></content>
+		</template>
+	</table>
+	<div class="d2l-table"><content></content></div>
+
+TODO: When ShadowDOM v1 is available, use <slot> and ::slotted
+-->
+<dom-module id="d2l-table-style">
+	<template>
+		<style include="d2l-colors">
+			.d2l-table {
```

Comment:
As for an HTML validator, it depends whether the validator uses the composed DOM or the light DOM. The Light DOM will be correct (since the div will be hidden). It is still yucky though